pytorch_lightning/callbacks/progress/base.py

```diff
@@ -105,7 +99,7 @@ def total_train_batches(self) -> int:
         return self.trainer.num_training_batches
 
     @property
-    def total_val_batches(self) -> int:
+    def total_val_batches(self) -> Union[int, float]:
```

Comment:
if float is due to inf, will it hold True for test and predict batches?

Reply:
Good eye, missed those, updated.